_posts/2023-16-15-Second-Issue-Newsletter

```diff
@@ -0,0 +1,292 @@
+* [Human Phenotype Ontology](https://obofoundry.org/ontology/hp.html) (HPO) is an ontology developed by the Monarch Initiative used to describe human phenotypic abnormalities seen in genetic disorders and clinical research. It provides a structured representation of abnormal characteristics associated with diseases. HPO helps researchers and clinicians share and integrate phenotypic data, making understanding and diagnosing genetic disorders easier. It uses a hierarchical organization and semantic relationships between terms and allows for annotation of genes and diseases. HPO is used in multiple diagnosis and variant prioritization tools, aiding healthcare professionals and researchers in identifying and classifying genetic conditions. Recently, HPO has released an international edition which covers different languages such as Chinese, Turkish, Japanese, Spanish and Czech ([https://obophenotype.github.io/hpo-translations/](https://obophenotype.github.io/hpo-translations/)). The translations are displayed in the HPO browser (e.g. [https://hpo.jax.org/app/browse/term/HP:0001166](https://hpo.jax.org/app/browse/term/HP:0001166)) and OLS ([https://www.ebi.ac.uk/ols4/ontologies/hp](https://www.ebi.ac.uk/ols4/ontologies/hp)).
+* [Cell Ontology (CL)](https://obofoundry.org/ontology/cl.html)  is an ontology designed to classify and describe cell types across different organisms. It serves as a resource for model organism and bioinformatics databases. The ontology covers a broad range of cell types in animal cells, with over 2700 cell type classes, and provides high-level cell type classes as mapping points for cell type classes in ontologies representing other species, such as the Plant Ontology or Drosophila Anatomy Ontology. Integration with other ontologies such as Uberon, GO, CHEBI, PR, and PATO enables linking cell types to anatomical structures, biological processes, and other relevant concepts. The Cell Ontology was created in 2004 and has been a core OBO Foundry ontology since the start of the Foundry. Since then, CL has been adopted by various efforts,  including the HuBMAP project, Human Cell Atlas (HCA), cellxgene platform, Single Cell Expression Atlas, BRAIN Initiative Cell Census Network (BICCN), ArrayExpress, The Cell Image Library, ENCODE, and FANTOM5, for annotating cell types and facilitating cellular reference mapping, as documented through various publications and examples.
+
+
+---
+
+
+## Members and Volunteers 
+
+
+### Member Spotlight:
```

Comment:
this is a singleton at h3 level, can be collapsed into the h2 level

Reply:
I think thats ok, sometimes this section has sub-sections, others it does.